README.md

```diff
@@ -1,5 +1,7 @@
 # eslint-config-fewlines-ts
 
+:warning: This package is deprecated in favor of [@fewlines/eslint-config](https://github.com/fewlinesco/eslint-config)
```

Comment:
I tried to open the preview but because we already have `:warning` two or three lines below, the message didn't trigger me.

What do you think of adding a more visible warning message like:

```suggestion
> :warning: **This package is deprecated in favor of [@fewlines/eslint-config](https://github.com/fewlinesco/eslint-config)**

---
```